File: C/calculadora_case.c
#include <stdio.h>

int main() {

  int num1,num2,resultado;
  int opcion;

  printf("Menu\n\n");
  printf("1- Suma\n");
  printf("2- Resta\n");
  printf("3- Multiplicacion\n");
  printf("4- Division\n");
  printf("Elige una opcion...: ");

  scanf("%i",&opcion);

  if (opcion > 0 && opcion < 5) {
    printf("Ingresa el primer elemento: ");
    scanf("%i",&num1);
    printf("Ingresa el segundo elemento: ");
    scanf("%i",&num2);
  }

  switch (opcion) {
  
      case 1:
        resultado = num1 + num2;
	printf("\nEl resultado de la operacion es: %i\n", resultado);
	break;
      case 2:
        resultado = num1 - num2;
	printf("\nEl resultado de la operacion es: %i\n", resultado);
	break;
      case 3:
        resultado = num1 * num2;
	printf("\nEl resultado de la operacion es: %i\n", resultado);
	break;
      case 4: 
	resultado = num1 / num2;
	printf("\nEl resultado de la operacion es: %i\n", resultado);
        break;
      default:
	printf("Opcion invalida\n");
	break;
  }
  
  return 0;

}




























 

File: Python/calculadora_case.py
import sys

def operaciones(opcion):
    num1 = int(input("Ingresa el primer elemento... "))
    num2 = int(input("Ingresa el segundo elemento... "))
    switcher = {
        1: num1 + num2,
        2: num1 - num2,
        3: num1 * num2,
        4: num1 / num2,
    }

    return switcher.get(opcion, "opcion invalida")

print("Menu")
print("1- Suma")
print("2- Resta")
print("3- Multiplicacion")
print("4- Division")
opcion = int(input("Elige una opcion..."))

if (opcion > 0 and opcion < 5):
    resultado = operaciones(opcion)
    print"El resultado de la operacion es: ", resultado
else:
    print("Opcion invalida")
